src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/GlobalSearchService.kt

```diff
@@ -61,6 +66,103 @@ class GlobalSearchService(
     }
   }
 
+  data class NomisContext(var nomisFinished: Boolean, var num: String?)
+
```

Comment:
This gets around the 10000 limit by using the 'scroll' api which should be able to get the whole index 10000 at a time